Packages/vcs/Lib/VTKPlots.py

```diff
@@ -406,9 +406,11 @@ def portrait(self, W, H, x, y, clear):
             self.renWin.SetSize(W, H)
 
     def initialSize(self):
-        # screenSize = self.renWin.GetScreenSize()
-        self.renWin.SetSize(self.canvas.bgX, self.canvas.bgY)
-        self._lastSize = (self.canvas.bgX, self.canvas.bgY)
+        screenSize = self.renWin.GetScreenSize()
+        bgX = int(screenSize[0]*.6)
```

Comment:
60% of the screen size, I thought it was obvious... That's a reasonable size but if you come up with a percentage you prefer feel free to update :wink: